pages/common/clj.md

```diff
@@ -0,0 +1,29 @@
+# clj
+
+> Clojure tool to start a REPL or invoke a specific function with data.
+> All options can be defined in a `deps.edn` file.
+> More information: <https://clojure.org/guides/deps_and_cli>.
+
+- Start a REPL:
+
+`clj [clj-opt*] [-A:aliases] [init-opt*]`
```

Comment:
Wow, this is a pretty complex example to be starting a page with. Are you aware of our `{{token_syntax}}` we've got documented in our [style guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md) at all? The page for `git` is a good example of how it's used.

The same goes for the rest of the page.

Reply:
oh, my bad. I haven't read that page.
I will change the example to just start a repl without options. that first one is the first example of `clj -h` but I agree it's confusing.

```suggestion
`clj`
```